docs/manual/format.md

```diff
@@ -2,13 +2,13 @@
 layout: default
 title: rpm.org - RPM Package format
 ---
+
 # Package format
 
-This document describes the RPM file format version 3.0, which is used
-by RPM versions 2.1 and greater.  The format is subject to change, and
-you should not assume that this document is kept up to date with the
-latest RPM code.  That said, the 3.0 format should not change for
-quite a while, and when it does, it will not be 3.0 anymore :-).
+This document describes the RPM file format version 4.0.  The format is subject
```

Comment:
Is "4.0" fine or ought we to use something along the lines of "V4" instead?

Reply:
This does not describe v4 file format, so we should not bump the number either. For v4, the by far most important item is the immutable region and that is not even mentioned anywhere here. Not surprising since most of the content is from the initial commit in 1996 (cdd4c29192131e1b6a3369b10f324928336a63da)

Reply:
I can add some descriptions of the immutable region.  I believe that's documented somewhere, maybe it's worth merging into this document.

Reply:
Done.